src/plugins/gui/providers/bityProvider.ts

```diff
@@ -293,7 +301,7 @@ const approveBityQuote = async (
   if (orderData.message_to_sign != null) {
     const { body } = orderData.message_to_sign
     const { publicAddress } = await wallet.getReceiveAddress({ tokenId: null })
-    const signedMessage = await wallet.signBytes(utf8.parse(body), { otherParams: { publicAddress } })
+    const signedMessage = await wallet.signMessage(body, { otherParams: { publicAddress } })
```

Comment:
This will re-break Etheruem because that engine's sign message throws if passed a hex message. Short term solution is to use signBytes for Ethereum and signMessage for bitcoin and litecoin.